docker/entrypoint.sh

```diff
@@ -22,6 +22,8 @@
 
 # A script to execute the tests inside the docker container.
 
+cd /github/workspace/src/main/cpp
+./build.sh
```

Comment:
i do not like this, since it would mean we have to build our c++ part every test.

do you know how much the overhead is?